modules/install-sno-installing-with-usb-media.adoc

```diff
@@ -0,0 +1,53 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="installing-with-usb-media_{context}"]
+
+= Installing with USB media
+
+Installing with USB media involves creating a bootable USB drive with the discovery ISO on the administration node. Booting the server with the USB drive prepares the node to install singe node OpenShift (SNO) via the Assisted Installer (AI).
+
+.Procedure
+
+. On the administration node, insert a USB drive into a USB port.
+
+. Create a bootable USB drive.
++
+[source,terminal]
+----
+# dd if=<path-to-iso> of=<path/to/usb> status=progress
+----
++
+For example:
++
+[source,terminal]
+----
+# dd if=discovery_image_sno.iso of=/dev/sdb status=progress
+----
++
+Once complete, the USB drive is ready for installation on the server.
```

Comment:
bit of a nit: "After the ISO is copied to the USB drive, you can use the USB drive to install <product-name>."